Command/ChapleanApiLogCleanCommand.php

```diff
@@ -34,14 +55,15 @@ protected function configure()
      * @param OutputInterface $output
      *
      * @return void
+     *
+     * @throws \Exception
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        /** @var ApiLogUtility $apiLogUtility */
-        $apiLogUtility = $this->getContainer()->get(ApiLogUtility::class);
+        $strDate = $input->getArgument('minimumDate') ?: 'now - 1 month midnight';
+        $dateLimit = new \DateTime($strDate);
```

Comment:
- [ ] Could be good to validate date and throw usefull exception message if not valid